builtin/logical/database/backend.go

```diff
@@ -125,6 +151,23 @@ type databaseBackend struct {
 	// concurrent requests are not modifying the same role and possibly causing
 	// issues with the priority queue.
 	roleLocks []*locksutil.LockEntry
+	// backendUUID is a unique identifier for this backend
+	backendUUID string
+}
+
+// addConnectionsCounter is used to keep metrics on how many and what types of databases we have open
+func (b *databaseBackend) addConnectionsCounter(dbw databaseVersionWrapper, amount int) {
```

Comment:
Would it be more accurate to call this something like `updateConnectionsGauge`?

Reply:
:+1: